_i18n/en/resources/user-guides/restore_account.md

```diff
@@ -1,53 +1,131 @@
 {% include disclaimer.html translated="no" translationOutdated="no" %}
 
-## Operating Systems:  Windows, Linux, Mac
+### Table of contents
 
-- Retrieve your 25 word @mnemonic-seed that you saved when creating your old Monero @wallet
+- [When should I restore a wallet?](#when-should-i-restore-a-wallet)
+- [What is needed to restore a wallet from mnemonic seed?](#what-is-needed-to-restore-a-wallet-from-mnemonic-seed)
+- [How to restore a wallet from mnemonic seed on Monero GUI](#how-to-restore-a-wallet-from-mnemonic-seed-on-monero-gui)
+- [How to restore a wallet from mnemonic seed on Monero CLI](#how-to-restore-a-wallet-from-mnemonic-seed-on-monero-cli)
+- [My restored wallet is showing a wrong balance](#my-restored-wallet-is-showing-a-wrong-balance)
 
-### Account Software:  monero-wallet-cli
+### When should I restore a wallet?
 
-- Open a command prompt and navigate to the drive and directory that contains monero-wallet-cli
+You should restore your @wallet in the following situations:
+- You lost access to your wallet (you forgot its password, your wallet file is corrupted or your device stopped working/is lost).
+- You want to create a copy of your wallet on another device.
 
-- At the command prompt type:  `monero-wallet-cli --restore-deterministic-wallet`
+This guide will teach you how to restore a Monero wallet from its @mnemonic-seed.
 
-- Once you press enter you will be prompted for a wallet file name.  Give your wallet a new name, any name will do
+If you want to restore a wallet from its private keys, please read [this guide]({{site.baseurl}}/resources/user-guides/restore_from_keys.html).
 
-- Press enter again and you will be prompted for a password.  Give your wallet a new and long password
+If you want to restore a wallet from a hardware wallet, please check [this page]({{site.baseurl}}/downloads/#hardware).
```

Comment:
This page doesn't give information about restoring a hardware wallet. I would clarify or remove it altogether.

Reply:
My original idea is to write a user guide on how to restore a wallet from a hardware wallet, but this guide doesn't exist yet. 

I could add a link to this non-existent page now and then we only merge this PR after the "how to restore a wallet from a hardware wallet" is created, or
I could remove it and then we add this link later, when the guide is created.

Reply:
We cannot merge it with a link to a document that doesn't exist yet. If that's the case, let's change this PR into a draft.

Reply:
Ok, I removed the link.